tools/vscode/ddlog-language/src/good/datstatements.json5

```diff
@@ -0,0 +1,46 @@
+{
+  "repository": {
+        "statements": {
+            "patterns": [
+                {
+                    "match": "(\\b)(?<=dump)(\\s+)([A-Z][a-zA-Z0-9_]*)(\\s*)",
```

Comment:
There is a significant amount of duplication in these two files (all command names are repeated), but I guess this works.